pip_audit/_dependency_source/pip.py

```diff
@@ -61,7 +65,9 @@ def collect(self) -> Iterator[Dependency]:
         # The `pip list` call that underlies `pip_api` could fail for myriad reasons.
         # We collect them all into a single well-defined error.
         try:
-            for (_, dist) in pip_api.installed_distributions(local=self._local).items():
+            for (_, dist) in pip_api.installed_distributions(
+                local=self._local, paths=list(self._paths)
```

Comment:
@di 
I think making the API take `List[os.PathLike]` was a mistake. Because of `List` being invariant, we have to go through this contortion to pass type checking. 😬 

Sorry, I was testing within the `pip-api` tests and calling it through the REPL so I didn't notice the typing issue. Should I make a PR to make this a `Sequence[os.PathLike]` instead?

Reply:
Sure!

Reply:
I'll merge as-is, and we can handle that with another PR 🙂